makefiles/Makefile.third_party.win.mk

```diff
@@ -17,6 +17,9 @@ SOPLEX_TAG = 2.2.0
 # Version of Sulum
 SULUM_TAG = 43
 
+# Added in support of clean third party targets
+TSVNCACHE_EXE = TSVNCache.exe
```

Comment:
I worry if people uses a different svn implementation that does not offer TSVNCACHE